tests/integration/test_contract.py

```diff
@@ -128,5 +132,19 @@ def test_contract_schema_validation(self):
             raise ValidationTestFailure("Msg should have failed validation") from exc
 
 
+class TestContractRestAPI(TestContract):
```

Comment:
Why not just use `NetworkConfig.fetchai_stable_testnet`?
Also in the other test file.

Reply:
it's grpc, but rest is needed here